src/brevitas_examples/llm/main.py

```diff
@@ -82,6 +80,8 @@ def validate(args):
             assert args.weight_quant_granularity == 'per_group', "Sharded torch group export requires per group weight quant."
             assert args.input_bit_width is None, "Sharded torch group weight export doesn't support input quant."
             assert not args.quantize_weight_zero_point, "Quantized weight zero point not supported."
+            if args.max_accumulator_bit_width is not None:
+                assert args.max_accumulator_tile_size == args.weight_group_size, "Group size must be equal to tile size."
```

Comment:
What if we want to use the entire row as tile?

Reply:
Updated and extended verification of CLI arguments.